going_postal.js

```diff
@@ -31,149 +31,153 @@ const fs = require('fs')
+			let c = numbers.length
+	
+			/*
+			Should compress the list even more, taking into account that places might
+			have postal numbers scattered around in the list. But somehow this makes
+			the final resulting file even larger.
+
+			This is probably due to that "sizeWithObject" computation fails badly. That
+			computation doesn't take into account the || expressions within the if.
+
+			C = (function() {
+				var tmp = 0
+				Object.keys(places).forEach(p => {
+					var newName = place.split('_')[0]
+					if(name === newName && places[p].length > 2) tmp += places[p].length
 			})
 
-			 return tmp
-		 })()*/
-
-		if(sizeWithIf(c, l) > sizeWithObject(c, l)) {
-			// Represent postal place sequence in a map, because this is the shortest
-			// representation.
-			numbers.forEach(number => {
-				singleBuf.push(parseInt(number) + ':"' + name + '"')
+				return tmp
+			})()*/
+	
+			if(sizeWithIf(c, l) > sizeWithObject(c, l)) {
+				// Represent postal place sequence in a map, because this is the shortest
+				// representation.
+				numbers.forEach(number => {
+					singleBuf.push(parseInt(number) + ':"' + name + '"')
+				})
+			} else {
+				// Represent postal place sequence in an if, because this is the shortest
+				// representation
+				let MIN = parseInt(numbers[0])
+				let MAX = parseInt(numbers[numbers.length - 1])
+	
+				postalIfMapping[name] = postalIfMapping[name] || []
+				postalIfMapping[name].push({MIN, MAX})
+			}
+		})
+	
+		buf.push('export default function postnummer(E) {')
+	
+		// Create possible OR statements if the postal place has several sequences
+		Object.keys(postalIfMapping).forEach(place => {
+			var exp = []
+			postalIfMapping[place].forEach(postals => {
+				exp.push('E>=' + postals.MIN + '&&E<=' + postals.MAX)
 			})
-		} else {
-			// Represent postal place sequence in an if, because this is the shortest
-			// representation
-			let MIN = parseInt(numbers[0])
-			let MAX = parseInt(numbers[numbers.length - 1])
-
-			postalIfMapping[name] = postalIfMapping[name] || []
-			postalIfMapping[name].push({MIN, MAX})
-		}
-	})
-
-	buf.push('export default function postnummer(E) {')
-
-	// Create possible OR statements if the postal place has several sequences
-	Object.keys(postalIfMapping).forEach(place => {
-		var exp = []
-		postalIfMapping[place].forEach(postals => {
-			exp.push('E>=' + postals.MIN + '&&E<=' + postals.MAX)
+			buf.push('if(' + exp.join('||') + ')return "' + place + '";')
 		})
-		buf.push('if(' + exp.join('||') + ')return "' + place + '";')
-	})
-
-	// The final map of postal number => postal place
-	buf.push('return ({' + singleBuf.join(',') + '})[E];')
-
-	buf.push('}')
-
-	fs.writeFile('./src/postal.js', buf.join(''), 'UTF-8', function() {
-
-	})
+	
+		// The final map of postal number => postal place
+		buf.push('return ({' + singleBuf.join(',') + '})[E];')
+	
+		buf.push('}')
+	
+		fs.writeFile('./src/postal.js', buf.join(''), 'UTF-8', () => {})
 })
```

Comment:
missing indentation

Reply:
Where do you mean?

Reply:
on the `})`

Reply:
Should be fixed 2a5ff6847e202b19896635ddc8d62d7428b43f10